test/e2e/apiextensions_mrds_test.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2025 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"testing"
+	"time"
+
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	"github.com/crossplane/crossplane/apis/apiextensions/v2alpha1"
+	"github.com/crossplane/crossplane/test/e2e/config"
+	"github.com/crossplane/crossplane/test/e2e/funcs"
+)
+
+func TestMRDValidation(t *testing.T) {
+	manifests := "test/e2e/manifests/apiextensions/mrd/validation"
+
+	cases := features.Table{
+		{
+			// A valid MRD should be created.
+			Name:        "ValidNewMRDIsAccepted",
```

Comment:
i swear i just made this comment in my review, but for some reason I'm not seeing it included 🤔 

Do we have e2e coverage for the scenario where a MRD is created (and active) and a resultant CRD is also created? i see we are checking that valid/invalid MRDs are accepted/rejected, but I don't also see e2e coverage for resulting CRDs being created and verified.